code/__DEFINES/mobs.dm

```diff
@@ -797,6 +797,12 @@ GLOBAL_LIST_INIT(layers_to_offset, list(
 /// This organ allows restyling with enamel restyling (like a fucking file or something?). It's for horns and shit
 #define EXTERNAL_RESTYLE_ENAMEL (1 << 3)
 
+// Bitflags to indicate which body markings we're allowed to use on the species
```

Comment:
"Bitflags"

These defines are not necessary anyways, just put the typepath in where it's used

Reply:
yeah idk why I even did that